requirements.txt

```diff
@@ -1,4 +1,4 @@
-OWSLib>=0.8.3
+OWSLib>=0.8.3,<0.19.0
```

Comment:
If the problem is only `Python 2` how about:

OWSLib>=0.8.3,<0.19.0; python_version <= '2'
OWSLib>=0.8.3; python_version > '2'

instead so we don't penalize Python 3 users?

Reply:
Argh, I hadn't considered this in my haste to unbreak the build from the OWSLib 0.19.0 changes.  We can create another PR so that Py3 users can use the latest version.

Reply:
Good idea, I'll create the PR  so we can get that in.